stable/external-dns/templates/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "external-dns.fullname" . }}
+  labels: {{ include "external-dns.labels" . | indent 4 }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+  strategy:
+    type: Recreate
```

Comment:
Is there an advantage to explicitly setting the strategy to `Recreate` rather than not providing one, thus defaulting to `RollingUpdate`?